src/docs/truffle/reference/contact-the-developers.md

```diff
@@ -22,7 +22,7 @@ The fastest way to get help is to reach out on our [community Gitter channel](ht
 
 ## Issue Tracker
 
-If you find a bug or want to recommend a new feature, the best way to reach us is by raising an issue on the GitHub project pages for [Truffle](https://github.com/trufflesuite/truffle), [Ganache](https://github.com/trufflesuite/ganache), or [Drizzle](https://github.com/trufflesuite/drizzle). If you're using another one of our tools or libraries, you may want to check out our [full list of GitHub projects](https://github.com/trufflesuite) to be sure you're raising your issue in the best place.
+If you find a bug or want to recommend a new feature, the best way to reach us is by raising an issue on the GitHub project pages for [Truffle](https://github.com/trufflesuite/truffle), [Ganache](https://github.com/trufflesuite/ganache-ui), or [Drizzle](https://github.com/trufflesuite/drizzle). If you're using another one of our tools or libraries, you may want to check out our [full list of GitHub projects](https://github.com/trufflesuite) to be sure you're raising your issue in the best place.
```

Comment:
Maybe we actually want to reference CL Ganache here?

Reply:
I think linking to UI still makes sense just because we don't have CLI documentation on the website right now; it is 99% Ganache UI focused.